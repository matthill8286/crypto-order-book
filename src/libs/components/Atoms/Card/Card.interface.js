
//# sourceMappingURL=Card.interface.js.map