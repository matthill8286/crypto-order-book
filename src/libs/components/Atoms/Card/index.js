export * from "./CardV2";
export * from "./Card.interface";
export { StyledCard } from "./Card.styled";
//# sourceMappingURL=index.js.map