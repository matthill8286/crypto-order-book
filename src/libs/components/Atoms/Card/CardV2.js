function ownKeys(object, enumerableOnly) { var keys = Object.keys(object); if (Object.getOwnPropertySymbols) { var symbols = Object.getOwnPropertySymbols(object); if (enumerableOnly) symbols = symbols.filter(function (sym) { return Object.getOwnPropertyDescriptor(object, sym).enumerable; }); keys.push.apply(keys, symbols); } return keys; }

function _objectSpread(target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i] != null ? arguments[i] : {}; if (i % 2) { ownKeys(Object(source), true).forEach(function (key) { _defineProperty(target, key, source[key]); }); } else if (Object.getOwnPropertyDescriptors) { Object.defineProperties(target, Object.getOwnPropertyDescriptors(source)); } else { ownKeys(Object(source)).forEach(function (key) { Object.defineProperty(target, key, Object.getOwnPropertyDescriptor(source, key)); }); } } return target; }

function _defineProperty(obj, key, value) { if (key in obj) { Object.defineProperty(obj, key, { value: value, enumerable: true, configurable: true, writable: true }); } else { obj[key] = value; } return obj; }

function _objectWithoutProperties(source, excluded) { if (source == null) return {}; var target = _objectWithoutPropertiesLoose(source, excluded); var key, i; if (Object.getOwnPropertySymbols) { var sourceSymbolKeys = Object.getOwnPropertySymbols(source); for (i = 0; i < sourceSymbolKeys.length; i++) { key = sourceSymbolKeys[i]; if (excluded.indexOf(key) >= 0) continue; if (!Object.prototype.propertyIsEnumerable.call(source, key)) continue; target[key] = source[key]; } } return target; }

function _objectWithoutPropertiesLoose(source, excluded) { if (source == null) return {}; var target = {}; var sourceKeys = Object.keys(source); var key, i; for (i = 0; i < sourceKeys.length; i++) { key = sourceKeys[i]; if (excluded.indexOf(key) >= 0) continue; target[key] = source[key]; } return target; }

function _extends() { _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

import * as React from 'react';
import { Badge } from "../Badge";
import { StyledBadgeLineWrapper, StyledCard, StyledCardWrapper, StyledDivider, StyledFeaturedAssetsContainer, StyledFeaturedAssetSpacing, StyledFeaturedAssetsPicture } from "./Card.styled";

var FeaturedAsset = function FeaturedAsset(_ref) {
  var image = _ref.image;
  return React.createElement(StyledFeaturedAssetSpacing, null, React.createElement(Card, {
    center: true,
    elevation: 1,
    display: "flex",
    shape: "rounded-small",
    padding: "xs",
    noBorder: ['bottom', 'top', 'right', 'left']
  }, React.createElement(StyledFeaturedAssetsPicture, {
    src: image,
    height: "32px",
    objectFit: "contain",
    width: "auto"
  })));
};

var FeaturedAssetRow = function FeaturedAssetRow(_ref2) {
  var floatingAssetRow = _ref2.floatingAssetRow,
      featuredAssetImages = _ref2.featuredAssetImages;

  if (featuredAssetImages && featuredAssetImages.length > 0) {
    var featuredAssets = featuredAssetImages.map(function (fa, index) {
      return React.createElement(FeaturedAsset, _extends({
        key: 'FeaturedAsset_' + fa.image + index
      }, fa));
    });
    return React.createElement(StyledFeaturedAssetsContainer, null, featuredAssets);
  } else {
    return null;
  }
};

export var Card = function Card(_ref3) {
  var ariaLabel = _ref3.ariaLabel,
      ariaLabelledby = _ref3.ariaLabelledby,
      children = _ref3.children,
      _ref3$center = _ref3.center,
      center = _ref3$center === void 0 ? false : _ref3$center,
      contentWidth = _ref3.contentWidth,
      _ref3$elevation = _ref3.elevation,
      elevation = _ref3$elevation === void 0 ? 0 : _ref3$elevation,
      elevationHover = _ref3.elevationHover,
      _ref3$fullHeight = _ref3.fullHeight,
      fullHeight = _ref3$fullHeight === void 0 ? false : _ref3$fullHeight,
      _ref3$noBorder = _ref3.noBorder,
      noBorder = _ref3$noBorder === void 0 ? [] : _ref3$noBorder,
      borderColor = _ref3.borderColor,
      _ref3$borderWidth = _ref3.borderWidth,
      borderWidth = _ref3$borderWidth === void 0 ? 1 : _ref3$borderWidth,
      _ref3$showDivider = _ref3.showDivider,
      showDivider = _ref3$showDivider === void 0 ? false : _ref3$showDivider,
      _ref3$floatingAssetRo = _ref3.floatingAssetRow,
      floatingAssetRow = _ref3$floatingAssetRo === void 0 ? false : _ref3$floatingAssetRo,
      dividerWidth = _ref3.dividerWidth,
      padding = _ref3.padding,
      margin = _ref3.margin,
      shape = _ref3.shape,
      _ref3$surface = _ref3.surface,
      surface = _ref3$surface === void 0 ? 'white' : _ref3$surface,
      _ref3$textColor = _ref3.textColor,
      textColor = _ref3$textColor === void 0 ? 'black' : _ref3$textColor,
      overflow = _ref3.overflow,
      _ref3$display = _ref3.display,
      display = _ref3$display === void 0 ? 'block' : _ref3$display,
      flexDirection = _ref3.flexDirection,
      _ref3$badges = _ref3.badges,
      badges = _ref3$badges === void 0 ? [] : _ref3$badges,
      _ref3$badgeActionType = _ref3.badgeActionType,
      badgeActionType = _ref3$badgeActionType === void 0 ? 'prominent' : _ref3$badgeActionType,
      onClick = _ref3.onClick,
      className = _ref3.className,
      featuredAssetImages = _ref3.featuredAssetImages,
      otherProps = _objectWithoutProperties(_ref3, ["ariaLabel", "ariaLabelledby", "children", "center", "contentWidth", "elevation", "elevationHover", "fullHeight", "noBorder", "borderColor", "borderWidth", "showDivider", "floatingAssetRow", "dividerWidth", "padding", "margin", "shape", "surface", "textColor", "overflow", "display", "flexDirection", "badges", "badgeActionType", "onClick", "className", "featuredAssetImages"]);

  var a11yProps = {
    'aria-label': ariaLabel,
    'aria-labelledby': ariaLabelledby,
    tabIndex: 0
  };
  var card = React.createElement(StyledCard, _extends({
    borderColor: borderColor,
    borderWidth: borderWidth,
    center: center,
    display: display,
    elevation: elevation,
    elevationHover: elevationHover,
    contentWidth: contentWidth,
    flexDirection: flexDirection,
    margin: margin,
    noBorder: noBorder,
    overflow: overflow,
    padding: padding,
    shape: shape,
    showDivider: showDivider,
    surface: surface,
    textColor: textColor
  }, onClick ? _objectSpread({}, a11yProps) : {}), children);
  return React.createElement(StyledCardWrapper, _extends({
    className: className,
    elevationHover: elevationHover,
    fullHeight: fullHeight,
    onClick: onClick
  }, onClick ? _objectSpread({}, a11yProps) : {}, otherProps), badges && badges.length > 0 ? React.createElement(React.Fragment, null, React.createElement(StyledBadgeLineWrapper, null, React.createElement(Badge, {
    badges: badges,
    isBadgeLine: true,
    actionType: badgeActionType
  })), card) : card, showDivider && React.createElement(StyledDivider, {
    dividerWidth: dividerWidth
  }), React.createElement(FeaturedAssetRow, {
    floatingAssetRow: floatingAssetRow,
    featuredAssetImages: featuredAssetImages
  }));
};
//# sourceMappingURL=CardV2.js.map