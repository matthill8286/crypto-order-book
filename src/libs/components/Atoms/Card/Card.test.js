import * as React from 'react';
import { media } from "../../../styles";
import { renderWithTheme } from "../../../testRenderer";
import { Card } from "./CardV2";
import { handlePadding } from "./Card.styled";
describe('Card', function () {
  var cardContent = 'Card ';
  it('Should render a Card', function () {
    var tree = renderWithTheme(React.createElement(Card, {
      elevation: 0
    }, cardContent));
    expect(tree).toMatchSnapshot();
  });
  it('Should render a Card with shape', function () {
    var tree = renderWithTheme(React.createElement(Card, {
      elevation: 0,
      shape: "sharp"
    }, cardContent));
    expect(tree).toMatchSnapshot();
  });
  it('Should render a Card with hover', function () {
    var tree = renderWithTheme(React.createElement(Card, {
      elevation: 0,
      elevationHover: 3
    }, cardContent));
    expect(tree).toMatchSnapshot();
  });
  it('Should render a Card with no border on right', function () {
    var tree = renderWithTheme(React.createElement(Card, {
      elevation: 0,
      noBorder: "right"
    }, cardContent));
    expect(tree).toMatchSnapshot();
  });
  it('Should render a Card with custom shape', function () {
    var tree = renderWithTheme(React.createElement(Card, {
      elevation: 0,
      shape: {
        bottomLeft: 'sharp',
        bottomRight: 'sharp',
        topLeft: 'rounded-small',
        topRight: 'rounded-small'
      }
    }, cardContent));
    expect(tree).toMatchSnapshot();
  });
  it('Should render a Card with padding, noBorder, shape, surface', function () {
    var tree = renderWithTheme(React.createElement(Card, {
      padding: {
        top: 'lg',
        bottom: 'lg',
        left: 'md',
        right: 'md'
      },
      elevation: 0,
      elevationHover: 0,
      noBorder: [],
      shape: {
        bottomLeft: 'sharp',
        bottomRight: 'sharp',
        topLeft: 'rounded-small',
        topRight: 'rounded-small'
      },
      surface: 'white'
    }, cardContent));
    expect(tree).toMatchSnapshot();
  });
  it('Should render a Card with elevation', function () {
    var tree = renderWithTheme(React.createElement(Card, {
      elevation: 1
    }, cardContent));
    expect(tree).toMatchSnapshot();
  });
  it('Should render a Card with surface', function () {
    var tree = renderWithTheme(React.createElement(Card, {
      elevation: 0,
      surface: "white"
    }, cardContent));
    expect(tree).toMatchSnapshot();
  });
  it('Should render a Card with padding', function () {
    var tree = renderWithTheme(React.createElement(Card, {
      elevation: 0,
      padding: "xxs"
    }, cardContent));
    expect(tree).toMatchSnapshot();
  });
  it('Should render a Card with custom padding', function () {
    var tree = renderWithTheme(React.createElement(Card, {
      elevation: 0,
      padding: {
        top: 'md',
        bottom: 'md',
        left: 'md',
        right: 'md'
      }
    }, cardContent));
    expect(tree).toMatchSnapshot();
  });
  it('Should render a Card with badges', function () {
    var tree = renderWithTheme(React.createElement(Card, {
      elevation: 0,
      badges: [{
        id: 0,
        name: 'New'
      }, {
        id: 1,
        name: '100% Free'
      }],
      badgeActionType: "prominent"
    }, cardContent));
    expect(tree).toMatchSnapshot();
  });
  it('Should render a Card with badges and hover effect', function () {
    var tree = renderWithTheme(React.createElement(Card, {
      elevation: 0,
      elevationHover: 4,
      shape: "rounded-small",
      display: "flex",
      badges: [{
        id: 0,
        name: 'New'
      }, {
        id: 1,
        name: '100% Free'
      }],
      badgeActionType: "prominent"
    }, cardContent));
    expect(tree).toMatchSnapshot();
  });
});
var themeSpacingBase = {
  xxs: '4px',
  xs: '8px',
  sm: '16px',
  md: '24px',
  lg: '32px',
  xl: '40px',
  xxl: '48px',
  xxxl: '56px',
  xxxxl: '64px'
};
describe('handlePadding(...)', function () {
  it('should return the padding for all sides (sm)', function () {
    var result = handlePadding(themeSpacingBase, 'sm');
    expect(result).toBe('padding: 16px;');
  });
  it('should return the padding only for top (sm)', function () {
    var padding = {
      top: 'sm'
    };
    var result = handlePadding(themeSpacingBase, padding);
    expect(result).toBe('padding-top: 16px;');
  });
  it('should return the padding only for right (lg)', function () {
    var padding = {
      right: 'lg'
    };
    var result = handlePadding(themeSpacingBase, padding);
    expect(result).toBe('padding-right: 32px;');
  });
  it('should return the padding only for bottom (xxs)', function () {
    var padding = {
      bottom: 'xxs'
    };
    var result = handlePadding(themeSpacingBase, padding);
    expect(result).toBe('padding-bottom: 4px;');
  });
  it('should return the padding only for left (xl)', function () {
    var padding = {
      left: 'xl'
    };
    var result = handlePadding(themeSpacingBase, padding);
    expect(result).toBe('padding-left: 40px;');
  });
  it('should return the padding only for left and top (xs, xxl)', function () {
    var padding = {
      left: 'xs',
      top: 'xxl'
    };
    var result = handlePadding(themeSpacingBase, padding);
    expect(result).toBe('padding-top: 48px;padding-left: 8px;');
  });
  it('should return separate paddings for all sides (top: md, right: sm, bottom: xs, left: md)', function () {
    var padding = {
      left: 'md',
      right: 'sm',
      top: 'md',
      bottom: 'xs'
    };
    var result = handlePadding(themeSpacingBase, padding);
    expect(result).toBe('padding-top: 24px;padding-right: 16px;padding-bottom: 8px;padding-left: 24px;');
  });
  it('should return a padding media-query for all sides (mobile|sm)', function () {
    var padding = {
      mobile: 'sm'
    };
    var result = handlePadding(themeSpacingBase, padding);
    expect(result).toBe(media.mobile + ' {padding: 16px; };');
  });
  it('should return a padding media-query for all sides (tablet|md)', function () {
    var padding = {
      tablet: 'md'
    };
    var result = handlePadding(themeSpacingBase, padding);
    expect(result).toBe(media.tablet + ' {padding: 24px; };');
  });
  it('should return a padding media-query for all sides (desktop|lg)', function () {
    var padding = {
      desktop: 'lg'
    };
    var result = handlePadding(themeSpacingBase, padding);
    expect(result).toBe(media.desktop + ' {padding: 32px; };');
  });
  it('should return a padding media-query for all sides (mobile|xs & desktop|xl)', function () {
    var padding = {
      mobile: 'xs',
      desktop: 'xl'
    };
    var result = handlePadding(themeSpacingBase, padding);
    expect(result).toBe(media.mobile + ' {padding: 8px; };' + media.desktop + ' {padding: 40px; };');
  });
  it('should return a padding media-query for all sides (mobile|tablet|desktop)', function () {
    var padding = {
      mobile: 'xxs',
      tablet: 'xl',
      desktop: 'xxl'
    };
    var result = handlePadding(themeSpacingBase, padding);
    expect(result).toBe(media.mobile + ' {padding: 4px; };' + media.tablet + ' {padding: 40px; };' + media.desktop + ' {padding: 48px; };');
  });
  it('should return a padding media-query for all sides but override top', function () {
    var padding = {
      mobile: 'xxs',
      tablet: 'xl',
      desktop: 'xxl',
      top: 'sm'
    };
    var result = handlePadding(themeSpacingBase, padding);
    expect(result).toBe(media.mobile + ' {padding: 4px; padding-top: 16px;};' + media.tablet + ' {padding: 40px; padding-top: 16px;};' + media.desktop + ' {padding: 48px; padding-top: 16px;};');
  });
  it('should return a padding media-query for all sides but override left', function () {
    var padding = {
      mobile: 'xxs',
      tablet: 'xl',
      desktop: 'xxl',
      left: 'md'
    };
    var result = handlePadding(themeSpacingBase, padding);
    expect(result).toBe(media.mobile + ' {padding: 4px; padding-left: 24px;};' + media.tablet + ' {padding: 40px; padding-left: 24px;};' + media.desktop + ' {padding: 48px; padding-left: 24px;};');
  });
  it('should return a padding media-query for all sides but override right and bottom', function () {
    var padding = {
      mobile: 'xxs',
      tablet: 'xl',
      desktop: 'xxl',
      right: 'sm',
      bottom: 'md'
    };
    var result = handlePadding(themeSpacingBase, padding);
    expect(result).toBe(media.mobile + ' {padding: 4px; padding-right: 16px;padding-bottom: 24px;};' + media.tablet + ' {padding: 40px; padding-right: 16px;padding-bottom: 24px;};' + media.desktop + ' {padding: 48px; padding-right: 16px;padding-bottom: 24px;};');
  });
});
//# sourceMappingURL=Card.test.js.map