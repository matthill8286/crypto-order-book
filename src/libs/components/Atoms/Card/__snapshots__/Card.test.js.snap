// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Should render a Card 1`] = `
<div
  className="Cardstyled__StyledCardWrapper-sc-1gxmhl7-6 jhkssp"
>
  <div
    className="Cardstyled__StyledCard-sc-1gxmhl7-1 dGwotJ"
    display="block"
    elevation={0}
  >
    Card 
  </div>
</div>
`;

exports[`Card Should render a Card with badges 1`] = `
<div
  className="Cardstyled__StyledCardWrapper-sc-1gxmhl7-6 jhkssp"
>
  <div
    className="Cardstyled__StyledBadgeLineWrapper-sc-1gxmhl7-0 cFLGQx"
  >
    <div
      className="Badgestyled__StyledBadgeLineWrapper-sc-1xo0r7x-1 ejODHn"
    >
      <div
        className="Badgestyled__StyledBadgeWrapper-sc-1xo0r7x-2 TMGmv"
      >
        <span
          className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 hycfbo Badgestyled__StyledTypo-sc-1xo0r7x-0 ePrcMg"
          fontSize="xxs"
        >
          <div
            className="Badgestyled__StyledOverflow-sc-1xo0r7x-3 iyMzXH"
          >
            New
          </div>
        </span>
      </div>
      <div
        className="Badgestyled__StyledBadgeWrapper-sc-1xo0r7x-2 TMGmv"
      >
        <span
          className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 hycfbo Badgestyled__StyledTypo-sc-1xo0r7x-0 ePrcMg"
          fontSize="xxs"
        >
          <div
            className="Badgestyled__StyledOverflow-sc-1xo0r7x-3 iyMzXH"
          >
            100% Free
          </div>
        </span>
      </div>
    </div>
  </div>
  <div
    className="Cardstyled__StyledCard-sc-1gxmhl7-1 dGwotJ"
    display="block"
    elevation={0}
  >
    Card 
  </div>
</div>
`;

exports[`Card Should render a Card with badges and hover effect 1`] = `
<div
  className="Cardstyled__StyledCardWrapper-sc-1gxmhl7-6 kFYWgF"
>
  <div
    className="Cardstyled__StyledBadgeLineWrapper-sc-1gxmhl7-0 cFLGQx"
  >
    <div
      className="Badgestyled__StyledBadgeLineWrapper-sc-1xo0r7x-1 ejODHn"
    >
      <div
        className="Badgestyled__StyledBadgeWrapper-sc-1xo0r7x-2 TMGmv"
      >
        <span
          className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 hycfbo Badgestyled__StyledTypo-sc-1xo0r7x-0 ePrcMg"
          fontSize="xxs"
        >
          <div
            className="Badgestyled__StyledOverflow-sc-1xo0r7x-3 iyMzXH"
          >
            New
          </div>
        </span>
      </div>
      <div
        className="Badgestyled__StyledBadgeWrapper-sc-1xo0r7x-2 TMGmv"
      >
        <span
          className="Typostyled__StyledInfoTypo-sc-1o9a6sy-0 hycfbo Badgestyled__StyledTypo-sc-1xo0r7x-0 ePrcMg"
          fontSize="xxs"
        >
          <div
            className="Badgestyled__StyledOverflow-sc-1xo0r7x-3 iyMzXH"
          >
            100% Free
          </div>
        </span>
      </div>
    </div>
  </div>
  <div
    className="Cardstyled__StyledCard-sc-1gxmhl7-1 sFHjI"
    display="flex"
    elevation={0}
    shape="rounded-small"
  >
    Card 
  </div>
</div>
`;

exports[`Card Should render a Card with custom padding 1`] = `
<div
  className="Cardstyled__StyledCardWrapper-sc-1gxmhl7-6 jhkssp"
>
  <div
    className="Cardstyled__StyledCard-sc-1gxmhl7-1 fLJaaG"
    display="block"
    elevation={0}
  >
    Card 
  </div>
</div>
`;

exports[`Card Should render a Card with custom shape 1`] = `
<div
  className="Cardstyled__StyledCardWrapper-sc-1gxmhl7-6 jhkssp"
>
  <div
    className="Cardstyled__StyledCard-sc-1gxmhl7-1 caHBET"
    display="block"
    elevation={0}
    shape={
      Object {
        "bottomLeft": "sharp",
        "bottomRight": "sharp",
        "topLeft": "rounded-small",
        "topRight": "rounded-small",
      }
    }
  >
    Card 
  </div>
</div>
`;

exports[`Card Should render a Card with elevation 1`] = `
<div
  className="Cardstyled__StyledCardWrapper-sc-1gxmhl7-6 jhkssp"
>
  <div
    className="Cardstyled__StyledCard-sc-1gxmhl7-1 ksdopm"
    display="block"
    elevation={1}
  >
    Card 
  </div>
</div>
`;

exports[`Card Should render a Card with hover 1`] = `
<div
  className="Cardstyled__StyledCardWrapper-sc-1gxmhl7-6 kFYWgF"
>
  <div
    className="Cardstyled__StyledCard-sc-1gxmhl7-1 iQbnEL"
    display="block"
    elevation={0}
  >
    Card 
  </div>
</div>
`;

exports[`Card Should render a Card with no border on right 1`] = `
<div
  className="Cardstyled__StyledCardWrapper-sc-1gxmhl7-6 jhkssp"
>
  <div
    className="Cardstyled__StyledCard-sc-1gxmhl7-1 fHRguC"
    display="block"
    elevation={0}
  >
    Card 
  </div>
</div>
`;

exports[`Card Should render a Card with padding 1`] = `
<div
  className="Cardstyled__StyledCardWrapper-sc-1gxmhl7-6 jhkssp"
>
  <div
    className="Cardstyled__StyledCard-sc-1gxmhl7-1 dUiCSJ"
    display="block"
    elevation={0}
  >
    Card 
  </div>
</div>
`;

exports[`Card Should render a Card with padding, noBorder, shape, surface 1`] = `
<div
  className="Cardstyled__StyledCardWrapper-sc-1gxmhl7-6 jhkssp"
>
  <div
    className="Cardstyled__StyledCard-sc-1gxmhl7-1 gifmKA"
    display="block"
    elevation={0}
    shape={
      Object {
        "bottomLeft": "sharp",
        "bottomRight": "sharp",
        "topLeft": "rounded-small",
        "topRight": "rounded-small",
      }
    }
  >
    Card 
  </div>
</div>
`;

exports[`Card Should render a Card with shape 1`] = `
<div
  className="Cardstyled__StyledCardWrapper-sc-1gxmhl7-6 jhkssp"
>
  <div
    className="Cardstyled__StyledCard-sc-1gxmhl7-1 iKFUGx"
    display="block"
    elevation={0}
    shape="sharp"
  >
    Card 
  </div>
</div>
`;

exports[`Card Should render a Card with surface 1`] = `
<div
  className="Cardstyled__StyledCardWrapper-sc-1gxmhl7-6 jhkssp"
>
  <div
    className="Cardstyled__StyledCard-sc-1gxmhl7-1 dGwotJ"
    display="block"
    elevation={0}
  >
    Card 
  </div>
</div>
`;
